### PR TITLE
TS-15-build-navigtion-flow

### PR DESCRIPTION
**Pull request by MuhammadARehman**
# jira
[TS-15](https://shop-go.atlassian.net/browse/TS-15)

description: 
- completed the navigation flow 

how to test it:
- fill the required info and run the application normally

### CODE REVIEW

File: ShopGo/CreateAccountExtension.swift [hunk 1]
```diff
@@ -89,15 +89,15 @@ extension  CreateScreenViewController : UITextFieldDelegate {
...
+                nextButton.alpha = 0.9
             }
         }
     }
     
     // Reset the storyboard
     func resetScreen() {
         nextButton.isEnabled = false
-        nextButton.alpha = 0.5
+//        nextButton.alpha = 0.9
```
[the remark this answers is not in the document]
Comment (MuhammadARehman) [reply ShopGo/CreateAccountExtension.swift:101]: sure thing

**Review by zuhair5 — DISMISSED** [review]
The story is good to be merged

**Review by haroonmaqsoodch — COMMENTED** [review]
Launch screen doesn't appear when the app is deleted and ran again

**Comment by MuhammadARehman** [comment]
> Launch screen doesn't appear when the app is deleted and ran again

That is due to the user default and is needed to be changed.if you want me to change let me know

**Review by haroonmaqsoodch — APPROVED** [review]

**Review by MuhammadHuzaifa2 — APPROVED** [review]

**Pull request merged**